src/cmd/manage_nsfs.js

```diff
@@ -945,6 +951,10 @@ function validate_options_type_by_value(input_options_with_data) {
             if (['allow_bucket_creation', 'regenerate', 'wide', 'show_secrets', 'force'].includes(option) && validate_boolean_string_value(value)) {
                 continue;
             }
+            // special case for booleans that can be unset (allows '')
+            if (option === 'force_md5_etag' && validate_boolean_string_value_with_unset(value)) {
+                continue;
+            }
```

Comment:
Why did you prefer adding a case for all the options (true, false, '') and not just the ''?
I'm suggestiong that in the previous condition add the `force_md5_etag` (the boolean true and false), and add a special case for the ''.
Having the two functions `validate_boolean_string_value` `validate_boolean_string_value_with_unset` might be confusing.
WDYT?